src/quantity.rs

```diff
@@ -210,6 +210,54 @@ macro_rules! quantity {
             impl super::Conversion<V> for super::$unit {})+
         }
 
+        #[cfg(feature = "std")]
+        mod parse {
+            use $crate::lib::str::FromStr;
+
+            /// Represents an error encountered while parsing a string into a `Quantity`.
+            #[derive(Clone, Copy, Debug)]
+            pub enum ParseError {
```

Comment:
This `enum` can be in `lib.rs` since it isn't system/quantity specific.

Reply:
I do like having it within `str`, although we could make a top-level `mod str`, I guess. What would you prefer?